app/widget/nodeview/nodeviewitem.cpp

```diff
@@ -367,22 +367,24 @@ void NodeViewItem::ReadjustAllEdges()
   }
 }
 
-void NodeViewItem::DrawNodeTitle(QPainter* painter, QString text, const QRectF& rect, Qt::Alignment vertical_align)
+void NodeViewItem::DrawNodeTitle(QPainter* painter, QString text, const QRectF& rect, Qt::Alignment vertical_align, bool draw_arrow)
 {
   QFontMetrics fm = painter->fontMetrics();
 
   painter->setRenderHint(QPainter::SmoothPixmapTransform);
 
   // Draw right or down arrow based on expanded state
-  int icon_size = fm.height() / 2;
-  int icon_padding = title_bar_rect_.height() / 2 - icon_size / 2;
-  int icon_full_size = icon_size + icon_padding * 2;
-  const QIcon& expand_icon = IsExpanded() ? icon::TriDown : icon::TriRight;
-  int icon_size_scaled = icon_size * painter->transform().m11();
-  painter->drawPixmap(QRect(title_bar_rect_.x() + icon_padding,
-                            title_bar_rect_.y() + icon_padding,
-                            icon_size,
-                            icon_size), expand_icon.pixmap(QSize(icon_size_scaled, icon_size_scaled)));
+  int icon_full_size = 0;
```

Comment:
Hmm wouldn't setting `icon_full_size` to 0 mess up the calculations that it's used for later on?

Reply:
I see what you mean, I forgot there'll always be an arrow if if this function hasn't drawn it. Moving the relevant calculations outside the if statement should do it right? Or will other arrow being a bit bigger cause problems?

Reply:
Ahh that's a good point, the size difference definitely makes it challenging. It might actually be fine as-is since the small subtitle should never be too wide for the node (though I guess we can't guarantee that through every language). Not sure what the best solution is in that case, outside of maybe passing along the arrow size to the function independently of the current font size.